modules/core/src/lib/deck.js

```diff
@@ -28,13 +28,20 @@ import log from '../utils/log';
 import GL from 'luma.gl/constants';
 import {AnimationLoop, createGLContext, trackContextState, setParameters} from 'luma.gl';
 import {Stats} from 'probe.gl';
-import {EventManager} from 'mjolnir.js';
+// import {EventManager} from 'mjolnir.js';
 
 import assert from '../utils/assert';
 import VENDOR_PREFIX from '../utils/css-vendor-prefix';
 import {EVENTS} from './constants';
 /* global document */
 
+let EventManager = null;
+try {
+  EventManager = require('mjolnir.js').EventManager;
```

Comment:
This is rather unfortunate - when using the `browser` field in package.json to support node, it does not cover the worker case.

Reply:
> This is rather unfortunate - when using the browser field in package.json to support node, it does not cover the worker case

Hmm, any way we can handle this inside mjolnir.js? In general, I think we want to strive for all of our libraries to be importable in all targets (browser, node, worker, ...). And the conditional reduction of functionality should ideally be done inside each library, rather than at import.